### PR TITLE
Migrate study-server under parent pom

### PR DESCRIPTION
**Pull request by etiennehomer**
Work in progress. Problem: exec jar is not generated.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -39,24 +39,18 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        
+        <maven.compiler.source>11</maven.compiler.source>
+    	<maven.compiler.target>11</maven.compiler.target>
 
-        <java.version>11</java.version>
```
Comment (jonenst) [inline pom.xml:43]: we need to keep this property

File: pom.xml [hunk 2]
```diff
@@ -73,45 +69,30 @@
                 <plugin>
                     <groupId>com.google.cloud.tools</groupId>
```
Comment (jonenst) [inline pom.xml:70]: you can remove this as it is already configured in powsybl-parent-ws

File: pom.xml [hunk 3]
```diff
@@ -73,45 +69,30 @@
...
                     <artifactId>jib-maven-plugin</artifactId>
-                    <version>${maven.jib.version}</version>
-                    <configuration>
-                        <from>
-                            <image>openjdk:11</image>
-                        </from>
-                    </configuration>
                 </plugin>
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
```
Comment (jonenst) [inline pom.xml:74]: you can remove this as it is already configured in powsybl-parent

File: pom.xml [hunk 4]
```diff
@@ -73,45 +69,30 @@
...
                     <artifactId>maven-clean-plugin</artifactId>
-                    <version>${maven.clean.version}</version>
                 </plugin>
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
```
Comment (jonenst) [inline pom.xml:78]: you can remove this as it is already configured in powsybl-parent

File: pom.xml [hunk 5]
```diff
@@ -73,45 +69,30 @@
...
                     <artifactId>maven-compiler-plugin</artifactId>
-                    <version>${maven.compiler.version}</version>
-                    <configuration>
-                        <release>${java.version}</release>
-                    </configuration>
                 </plugin>
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
```
Comment (jonenst) [inline pom.xml:82]: you can remove this as it is already configured in powsybl-parent

File: pom.xml [hunk 6]
```diff
@@ -73,45 +69,30 @@
...
                     <artifactId>maven-install-plugin</artifactId>
-                    <version>${maven.install.version}</version>
                 </plugin>
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
```
Comment (jonenst) [inline pom.xml:86]: you can remove this as it is already configured in powsybl-parent

File: pom.xml [hunk 7]
```diff
@@ -73,45 +69,30 @@
...
                     <artifactId>maven-site-plugin</artifactId>
-                    <version>${maven.site.version}</version>
                 </plugin>
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
```
Comment (jonenst) [inline pom.xml:90]: you can remove this as it is already configured in powsybl-parent

File: pom.xml [hunk 8]
```diff
@@ -73,45 +69,30 @@
...
                     <artifactId>maven-surefire-plugin</artifactId>
-                    <version>${maven.surefire.version}</version>
                 </plugin>
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
```
Comment (jonenst) [inline pom.xml:94]: you can remove this as it is already configured in powsybl-parent

File: pom.xml [hunk 9]
```diff
@@ -121,36 +102,10 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-checkstyle-plugin</artifactId>
-                <version>${maven.checkstyle.version}</version>
```
Comment (jonenst) [inline pom.xml:124]: you can remove this as it is already configured in powsybl-parent

File: pom.xml [hunk 10]
```diff
@@ -121,36 +102,10 @@
...
-                            <consoleOutput>true</consoleOutput>
-                            <failsOnError>true</failsOnError>
-                            <includeTestSourceDirectory>true</includeTestSourceDirectory>
-                        </configuration>
-                    </execution>
-                </executions>
             </plugin>
             <plugin>
                 <groupId>org.springframework.boot</groupId>
                 <artifactId>spring-boot-maven-plugin</artifactId>
```
Comment (jonenst) [inline pom.xml:108]: you can remove this as it is already configured in powsybl-parent-ws

File: pom.xml [hunk 11]
```diff
@@ -166,28 +121,6 @@
                     <plugin>
                         <groupId>org.jacoco</groupId>
                         <artifactId>jacoco-maven-plugin</artifactId>
-                        <version>${maven.jacoco.version}</version>
```
Comment (jonenst) [inline pom.xml:169]: you can remove this as it is already configured in powsybl-parent

File: pom.xml (hunk 10 above)
Comment (etiennehomer) [reply pom.xml:108]: Are you sure for this one ? The exec jar isn't generated when I remove the spring-boot-maven-plugin

File: pom.xml [hunk 12]
```diff
@@ -306,12 +193,6 @@
             <version>${embedded-cassandra.version}</version>
             <scope>test</scope>
         </dependency>
-        <dependency>
```
Comment (jonenst) [inline pom.xml:309]: you need this

**Pull request merged**